miner/src/pool/queue.rs

```diff
@@ -179,8 +178,14 @@ impl TransactionQueue {
 
 		let verifier = verifier::Verifier::new(client, options, self.insertion_id.clone());
 		let results = transactions
-			.into_par_iter()
-			.map(|transaction| verifier.verify_transaction(transaction))
+			.into_iter()
+			.map(|transaction| {
+				if self.pool.read().find(&transaction.hash()).is_some() {
+					bail!(transaction::Error::AlreadyImported)
+				}
```

Comment:
Was the primary cause of the high CPU usage the parallel verification or verifying already imported txs. Or a combination of both?

Reply:
So it seems like a combination of even more things, this brings the load down, but not 100% to the level of the one of 1.10. I'm still figuring out other culprits.